client/script.js

```diff
@@ -0,0 +1,532 @@
+    modalButtonsDiv.setAttribute('class', 'modal__buttons');
+
+    const closeModalButton = document.createElement('button');
+    closeModalButton.setAttribute('type', 'button');
+    closeModalButton.setAttribute('class', 'button close-modal');
+    closeModalButton.textContent = 'Cancel';
+
+    const saveModalButton = document.createElement('button');
+    saveModalButton.setAttribute('type', 'button');
+    saveModalButton.setAttribute('class', 'button button--pink');
+    saveModalButton.textContent = 'Save';
+
+    modalDiv.appendChild(modalContent);
+    modalContent.appendChild(modalTitle);
+    modalContent.appendChild(inputsContainer);
+    inputsContainer.appendChild(input1);
+    inputsContainer.appendChild(input2);
+    inputsContainer.appendChild(input3);
+    inputsContainer.appendChild(input4);
+    inputsContainer.appendChild(input5);
+    inputsContainer.appendChild(input6);
+    modalContent.appendChild(textarea);
+    modalContent.appendChild(modalButtonsDiv);
+    modalButtonsDiv.appendChild(closeModalButton);
+    modalButtonsDiv.appendChild(saveModalButton);
+
+    return modalDiv;
+}
+
+// RENDER THE MODAL
+function renderModal() {
+    const domModal = createModal();
+    modal.appendChild(domModal);
+    createModal();
+}
+
+renderModal();
+
+// CLEAR THE CONTENT
+function clearRoot() {
+    root.innerHTML = '';
+}
+
+// CREATE HASH ROUTE
+function locationHashChange(articles) {
+    (location.hash === '#/') && renderArticle(articles);
+    (location.hash.includes('#/article')) && renderAllArticlesDetails(articles);
+}
+
+
+// GETTING THE ELEMENTS TO CLOSE THE MODAL
+let modalOverlay = document.querySelector(".modal__overlay");
+let closeModal = document.querySelector(".close-modal")
+
+// OPEN MODAL FUNCTION, called directly in the function that creates the ADD BUTTON
+function openModal() {
+    modalOverlay.style.visibility = "visible";
+    modalOverlay.style.opacity = 1;
+}
+
+// CLOSING THE MODAL
+closeModal.addEventListener("click", function() {
+    modalOverlay.style.visibility = "hidden";
+    modalOverlay.style.opacity = 0;
+})
+
+
+// DARK MODE
+const toggleSwitch = document.querySelector('.theme-switch input[type="checkbox"]');
+let body = document.querySelector('body');
+
+function switchTheme(e) {
+    if (e.target.checked) {
+        body.setAttribute('data-theme', 'dark');
+        localStorage.setItem('theme', 'dark'); //add this
+    } else {
+        body.setAttribute('data-theme', 'light');
+        localStorage.setItem('theme', 'light'); //add this
+    }
+}
+
+
+toggleSwitch.addEventListener('change', switchTheme, false);
+
+const currentTheme = localStorage.getItem('theme') ? localStorage.getItem('theme') : null;
+
+if (currentTheme) {
+    body.setAttribute('data-theme', currentTheme);
+
+    if (currentTheme === 'dark') {
+        toggleSwitch.checked = true;
+    }
+}
+
+// DELETING ARTICLE DEPENDING ON THE ID, function called directly where the delete button is created (createArticle)

```

Comment:
Great, you implemented delete method! 💯

Reply:
I appreciate the fact that you wrote comments for every function. The code is very easy to understand